package.json

```diff
@@ -73,7 +74,7 @@
     "build": "./node_modules/.bin/tsc",
     "start": "./node_modules/.bin/tsc -w",
     "test": "jest test",
-    "test-with-coverage": "jest test --coverage --collectCoverageFrom '**/*.{ts,tsx}'",
+    "test-with-coverage": "jest test --coverage --collectCoverageFrom 'src/**/*.ts'",
```

Comment:
`test-with-coverage` was failing for me, and I don't think there are any React `.tsx` files, so I changed this to only look at `.ts` files in the `src` directory.